README.md

```diff
@@ -432,4 +428,4 @@ To completely remove the scheduled export setup:
 python setup_launchd.py --remove
 ```
 
-This unloads the job and removes the scheduling files while leaving your main `exportnotes.zsh` script and any exported data intact.
\ No newline at end of file
+This unloads the job and removes the scheduling files while leaving your main `exportnotes.zsh` script and any exported data intact.
```

Comment:
I didn't change anything here intentionally; guessing this is just the GitHub web code editor doing something with a newline at the end of the doc